site/update-image.md

```diff
@@ -109,6 +110,19 @@ The `latestPSU` option will continue to be supported for the CREATE option, but
 UPDATE option.  Because of the number of patches and their size, using `latestPSU` as an update to an existing image can 
 increase the size of the image significantly, and is not recommended. 
 
+#### Resource Template Files
+
+If provided, the file or files provided with `--resourceTemplates` will be overwritten. For known tokens, 
+the placeholders will be replaced with values according to the table below.  **Note:** Placeholders must follow
```

Comment:
according to the table below -> according to the following table

Reply:
Thank you, fixed.